### DISCUSSION TITLE
Distribute Nornir applciations

### DISCUSSION THREAD

**Discussion by Dylan-BDNS** [nornir-automation/nornir#753] [Q&A]

**Comment by dbarrosop** [comment]
I am afraid I don't understand your issue, you will have to expand on that. In any case, there is no reason why nornir shouldn't be able to be packaged by standard packaging tools, I'd recommend docker images though.

**Comment by dmulyalin** [comment]
If you willing to pack nornir in containers can have a look at salt-nornir-sdocker as example, remove the salt parts and what will be left is Nornir with a number of plugins. https://github.com/dmulyalin/salt-nornir-docker
